.github/workflows/python-test.yaml

```diff
@@ -37,5 +37,7 @@ jobs:
             pip install -r requirements-dev.txt
         - name: Lint with ruff
           run: ruff .
+        - name: Check formatting with black
```

Comment:
So . is the root directory in this case?

Reply:
Yes, so that it includes the Python files in both scripts and backend.